### DISCUSSION TITLE
How to use Delta bundles or multiple bundles

### DISCUSSION THREAD

**Discussion by vas-rus** [open-policy-agent/community#240] [Community]
Hi Team, 
I am solving a problem that along with data.json of policy there are additional fields from user configuration that I want to be available for rego file processing. I found that Delta bundles can be used for same. Can someone guide me how to create config and use delta bundle for same. The document currently doesn't have this option for those, who are just starting with opa. 
A simple config along with delta bundle will be helpful. 
Moreover there is written one more way like configuring two bundles in "roots". If someone can put light there also.

**Comment by peteroneilljr** [comment]
Hi @vas-rus 

Can you provide a small data example and the end result you want it to look like? Then I can provide an example of a Delta bundle patch.

Cheers,
Peter
